src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/controllers/web/GitController.kt

```diff
@@ -0,0 +1,21 @@
+package org.vaccineimpact.orderlyweb.controllers.web
+
+import org.vaccineimpact.orderlyweb.ActionContext
+import org.vaccineimpact.orderlyweb.controllers.Controller
+import org.vaccineimpact.orderlyweb.getFriendlyDateTime
+import org.vaccineimpact.orderlyweb.models.GitCommit
+import java.time.LocalDateTime
+
+class GitController(context: ActionContext): Controller(context)
+{
+    fun getCommits() : List<GitCommit>
+    {
+        //TODO: Get from Orderly server when supported
+        return listOf(
+               GitCommit("d41cc19", "2020-06-02T08:51:251",
```

Comment:
The date format coming out of orderly server is `2020-06-10 10:12:04` i.e. `yyyy-MM-dd HH:mm:ss`

Reply:
Oh from looking at HelperTests this format might already by dealt with.

Reply:
The data coming back from the endpoint also includes an age in seconds. This is the time since the commit was made.

Data looks like

|id            |date_time                |age|
|------------|---------------------------|-----|
|854cd4a|2020-06-10 10:12:04|2    |
|460c325|2020-06-10 10:12:01|5    |
|1ca88ef |2020-06-10 10:12:00|6    |

Expecting that if we think age is a more useful thing to display (similar to how git shows age) we could convert into some appropriate unit for displaying.

Reply:
Cool, I'll update the dummy data to reflect the real data and see how it copes. 

Will see how easy it is to put in 'X units ago' string, would be nice to make it git-like.